tutorials/scripting/nodes_and_scene_instances.rst

```diff
@@ -0,0 +1,205 @@
+.. _doc_nodes_and_scene_instances:
+
+Nodes and scene instances
+=========================
+
+This guide explains how to get nodes, create nodes, add them as a child, and
+instantiate scenes from code.
+
+Getting nodes
+-------------
+
+You can get a reference to a node by calling the :ref:`Node.get_node()
+<class_Node_method_get_node>` method. For this to work, the child node must be
+present in the scene tree. Getting it in the parent node's ``_ready()`` function
+guarantees that.
+
+Say you have a scene tree like this, and you want to get a reference to the
+Sprite and Camera2D nodes to access them in your script.
+
+.. image:: img/nodes_and_scene_instances_player_scene_example.png
+
+To do so, you can use the following code.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    var sprite
+    var camera2d
+
+    func _ready():
+        sprite = get_node("Sprite")
+        camera2d = get_node("Camera2D")
+
+ .. code-tab:: csharp
+
+    private Sprite _sprite;
+    private Camera2D _camera2d;
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        _sprite = GetNode<Sprite>("Sprite");
+        _camera2d = GetNode<Camera2D>("Camera2D");
+    }
+
+Note that you get nodes by their name. For example, if you rename the Sprite
+node into Skin, the call to get it would have to be ``get_node("Skin")``.
+
+Node paths
+----------
+
+You're not limited to getting a direct child. The ``get_node()`` function
+supports paths, a bit like when working with a file browser. Add a slash to
+separate nodes.
+
+Take the following example scene, with the script attached to the UserInterface
+node.
+
+.. image:: img/nodes_and_scene_instances_ui_scene_example.png
+
+To get the Tween node, you would use the following code.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    var tween
+
+    func _ready():
+        tween = get_node("ShieldBar/Tween")
+
+ .. code-tab:: csharp
+
+    private Tween _tween;
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        _tween = GetNode<Tween>("ShieldBar/Tween");
+    }
+
+.. note:: As with file paths, you can use ".." to get a parent node. The best
+          practice is to avoid doing that though not to break encapsulation and
+          keep your code organized. You can also start the path with a forward
```

Comment:
```suggestion
          practice is to avoid doing that though not to break encapsulation.
          You can also start the path with a forward
```